spec/controllers/boards_controller_spec.rb

```diff
@@ -61,16 +61,12 @@
   end
 
   describe "POST create" do
-    #let!(:user) {create(:user)}
-    #let(:params) {{board: { title: 'New title 123'}}}
-    # let(:call) {post :create, params}
+    let(:params) {{board: { title: 'New title 123'}}}
+    let(:call) {post :create, params}
     
     it "assigns owner" do
-      # post :create, params
-      # expect(Board.last.owner).to eq(user)
-
-      board_params = FactoryGirl.attributes_for(:board)
-      expect { post :create, board: board_params }.to change(Board, :count).by(1) 
+      call
```

Comment:
Staraj sie w it miec zawsze tylko expect ...sa wyjatki ale jak sie da to tak robimy :) merguj pr'a